tardis/base.py

```diff
@@ -12,10 +12,11 @@ def run_tardis(
     packet_source=None,
     simulation_callbacks=[],
     virtual_packet_logging=False,
-    show_convergence_plots=True,
+    show_convergence_plots=None,
     log_level=None,
     specific_log_level=None,
-    show_progress_bars=True,
+    show_progress_bars=None,
+    tardis_gui=False,
```

Comment:
You shouldn't really add another option - we have so many already. You can just set default values of both `show_convergence_plots` and `show_progress_bars` to False

Reply:
Done!